test/TestCaseReader.h

```diff
@@ -31,6 +32,7 @@ namespace solidity::frontend::test
 struct SourceMap
 {
 	std::map<std::string, std::string> sources;
+	std::vector<std::pair<std::string, std::string>> externalSources;
```

Comment:
When looking at this at first I thought you're storing the source in both and wanted to comment that you could just store the target for external sources. But I see that's what you're doing. It would be much more apparent if you used `boost::filesystem::path` here:
```suggestion
	std::vector<std::pair<std::string, boost::filesystem::path>> externalSources;
```